AndroidLibV2rayLite/Process/Escort/escort.go

```diff
@@ -1,11 +1,10 @@
 package Escort
 
 import (
+	"log"
 	"os"
 	"os/exec"
 
-	"log"
```

Comment:
could you try to clean this outdated commit, did you really based on latest upstream?